dev-libs/dotconf/dotconf-1.3-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
```

Comment:
You can get rid of the `eutils` eclass but you must inherit `toolchain-funcs` for the `tc-getCC` to work properly.

Reply:
Done.